framework_tvb/tvb/core/entities/file/files_helper.py

```diff
@@ -99,6 +99,13 @@ def get_project_folder(self, project, *sub_folders):
             self.check_created(complete_path)
         return complete_path
 
+    @staticmethod
+    def get_key_by_value(list, value):
```

Comment:
Why this static method in FilesHelper? it doe not seem to be related to files.
Maybe put it as a function in `tvb.core.utils` ?